NOTIFY/events.py

```diff
@@ -117,17 +135,16 @@ def reminder(self):
             for names in birthday_list:
                 birthday_msg = random.choice(birthday_greeting).format(names)
                 birthday_img = None
-                chat = pc(self.mode)
                 status_code = chat.send_message(birthday_msg,birthday_img)
-                print('Status code = {}'.format(status_code))
+                print('Status code = {}, birthday message for today sent!'.format(status_code))
 
-        
-    
     
 if __name__ == '__main__':
-    mode = 'personal'
+    mode = 'ben'
     today = datetime.datetime.today()
-    ev = Event(today, mode)
+    chatter = pc(mode)
+    ev = Event(today, chatter)
+    if today.hour < 12:
```

Comment:
Berati disini sengerti gua ada run scriptnya bakal 3 kalian ya?
1. Pagi di bawah jam 12 bakal eksekusi renungan pagi, sama reminder2
2. Siang jam 12 buat renungan desiring god
3. Malam sisanya buat renungan malam?

Reply:
Iya, tapi ini cmn ide doang. Enaknya gmn? Aku tadi cuman  kepikiran biar ga numpuk gt jadi di 3 waktu berbeda, tp open for suggestions sih gmn gt2nya hehe.

Aku kepikiran kita bisa bikin timeline posting an nya gt jadi bs atur kapan post renungan, ultah, link article, reminder event gt2

Reply:
Iya, gua juga jujur rada ga ada ide, tapi kalo post tiap beberapa kali annoying juga ga sih? Kaya ribut banget chat nya. Itu aja lah kalo ga